### PR TITLE
Resolving build test issues in PRs

### PR DESCRIPTION
**Pull request by jmcshane**
# Changes

Resolving small build test issues

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#tests) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#docs) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commits)
- [ ] Release notes block has been filled in or deleted (only if no user facing changes)

_See [the contribution guide](https://github.com/tektoncd/triggers/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Comment by jmcshane** [comment]
@tektoncd/triggers-maintainers this PR just has a single newline and were getting issues with the build tests. im going to change this PR into one that fixes all these issues

**Comment by jmcshane** [comment]
Here's my empty line commit build test: https://prow.tekton.dev/log?job=pull-tekton-triggers-build-tests&id=1390512994986758147

File: pkg/reconciler/v1alpha1/eventlistener/eventlistener.go [hunk 1]
```diff
@@ -613,7 +613,7 @@ func (r *Reconciler) reconcileCustomObject(ctx context.Context, logger *zap.Suga
 			}
 		}
 
-		//if dynamicduck.ReconcileCustomObject(existingCustomObject, data) {
+		// if dynamicduck.ReconcileCustomObject(existingCustomObject, data) {
```
Comment (dibyom) [inline pkg/reconciler/v1alpha1/eventlistener/eventlistener.go:616]: Should we remove this comment?

Comment (jmcshane) [reply pkg/reconciler/v1alpha1/eventlistener/eventlistener.go:616]: maybe that's a separate PR...not sure the context behind the comment

**Pull request merged**